Sources/Tracking/Service/Request/AddUpdateCustomerRequestBody.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
```

Comment:
maybe I missed it, but are we not actually using this?

also this isn't correct, the body is just a json object but there are specific well-known fields:
- email
- _update
- anonymous_id
- created_at

but they will get serialized into the main object body
```json
{
  "email": "abc@example.com",
  "_update": true,
  "anonymous_id": "RIOEJOIMLW",
  "created_at": 1629854504,
  "t_shirt_size": "M",
  "premium": true,
  "ltv": 123400
}
```
It wouldn't be so bad to work with a struct and then do some custom serialization on it to make this happen, makes the fields more explicit. But we should be careful here to match the semantics of the API